packages/commonwealth/client/scripts/views/components/component_kit/new_designs/CWUpvoteSmall.tsx

```diff
@@ -44,7 +46,8 @@ const CWUpvoteSmall = ({
               onMouseLeave={handleInteraction}
             >
               <CWThreadAction
-                action="upvote"
+                action= "upvote"
```

Comment:
Spacing

Reply:
this should not be a case if prettier is correctly steup, but it seems that it is not on your machine @Miaplacidus ?

https://commonxyz.slack.com/archives/C050AE0URFH/p1689075560213459